schemas/blocks/relationship_collector.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["RelationshipCollector"]
+      },
+      "title": {
+        "type": "string"
+      },
+      "list": {
```

Comment:
Maybe `list_name` or something like that?. I think we try to refrain using python built-in types so the python codes are not confusing etc.